.github/workflows/main.yml

```diff
@@ -7,7 +7,11 @@ jobs:
         runs-on: ${{ matrix.os }}
         strategy:
             matrix:
-                os: [ubuntu-18.04, ubuntu-16.04, windows-latest, macos-latest]
+                os: 
+                    - ubuntu-18.04
+                    - ubuntu-16.04
+                # - windows-latest
```

Comment:
Currently, GHC 8.6.5 not found on macOS or Windows with setup-haskell.
Ref: https://github.com/actions/setup-haskell/issues/1